frontend/shared/src/components/header/Header.tsx

```diff
@@ -17,13 +17,12 @@ export type HeaderProps = {
   titleUrl?: string;
   skipToContentLabel?: string;
   menuToggleAriaLabel?: string;
-  locale: string;
-  languages: OptionType<string>[];
+  languages?: OptionType<string>[];
```

Comment:
some services (as handler) does not support translations so let's make languages optional